ocaml/backends/verilog.ml

```diff
@@ -227,6 +227,14 @@ let assignment_to_string' (gensym: int ref) (assignment: assignment) =
    | EUnop (fn, arg1) ->
       (match fn with
        | Not _ -> default_left ^ "~" ^ arg1
+       | Rev sz ->
+        let rec loop n = if n = 0
+          then Printf.sprintf "%s[%d]" arg1 n
+          else Printf.sprintf "%s[%d], " arg1 n ^ loop (n - 1) in
+        let rhs = if sz = 0
+            then "{}" (* is that legal for a bits 0 vector? *)
+            else "{" ^ loop (sz-1) ^ "}" in
```

Comment:
For `n = 0` this should be enough: `x = y`. There is no real vector. So it should not need indexing.

Reply:
It  should be true for `n = 1` as well.